src/Sentry.Bindings.Android/Sentry.Bindings.Android.csproj

```diff
@@ -12,6 +12,7 @@
   <ItemGroup>
     <None Remove="$(MSBuildThisFileDirectory)..\..\README.md" />
     <None Include="$(MSBuildThisFileDirectory)README.md" Pack="true" PackagePath="" />
+    <None Include="$(MSBuildThisFileDirectory)build/Sentry.Bindings.Android.targets" Pack="true" PackagePath="build/Sentry.Bindings.Android.targets" />
```

Comment:
@bitsandfoxes I had to add this to prevent [this warning](https://github.com/getsentry/sentry-dotnet/actions/runs/10822485750/job/30026411627?pr=3513):
> error NU5129: Warning As Error: - At least one .targets file was found in 'build/', but 'build/Sentry.Bindings.Android.targets' was not. 

It looks like the NuGet packager expects the targets file to be in the `build/` directory whereas our project file copies/copied it into the `build/$(TargetFramework)$(TargetPlatformVersion)/` folder.

For the time being, I've left it being copied to both destinations since I don't have enough context to know/understand why it was originally being copied to a TargetFramework/PlatformVersion specific folder... something to do with proguard, I'm assuming???